### PR TITLE
Add a `simtools-simulate-prod` integration test for the South site

### PR DESCRIPTION
**Pull request by GernotMaier**
This adds a single integration tests for `simtools-simulate-prod` for the South site to make sure that export of the southern telescope types works.

These tests are still missing the validation of the generated CORSIKA /sim_telarray log files (see issue #882).

### CODE REVIEW

File: tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ CTA_SIMPIPE:
   APPLICATION: simtools-simulate-prod
   TEST_NAME: gamma_20_deg
   CONFIGURATION:
-    PRODUCTION_CONFIG: ./tests/resources/prod_multi_config_test.yml
+    PRODUCTION_CONFIG: ./tests/resources/prod_multi_config_test_North.yml
     MODEL_VERSION: prod6
     SITE: North
     PRIMARY: gamma
```
Comment (tobiaskleiner) [inline tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml:8]: no integration_tests for north here?

**Review by tobiaskleiner — COMMENTED** [review]
Just one comment about the lack of intgration_tests configuration in the `..20_deg_North.yml `file.

File: tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml (hunk 1 above)
Comment (VictorBarbosaMartins) [reply tests/integration_tests/config/simulate_prod_gamma_20_deg_North.yml:8]: There is one test implemented, but in the other PR: https://github.com/gammasim/simtools/pull/875

**Comment by VictorBarbosaMartins** [comment]
> Just one comment about the lack of intgration_tests configuration in the `..20_deg_North.yml `file.

Thanks @tobiaskleiner ! Please let me know if there is something else here.
In any case, tests will fail for the moment because of the model parameters validation.

**Comment by VictorBarbosaMartins** [comment]
@tobiaskleiner everything looks good here now. Would you please have another look before approving?

**Review by tobiaskleiner — APPROVED** [review]

**Comment by VictorBarbosaMartins** [comment]
Great, thank you!

**Pull request merged**